test/tools/runner/config.ts

```diff
@@ -294,7 +294,23 @@ export class TestConfiguration {
    *
    * @param options - overrides and settings for URI generation
    */
-  url(options?: UrlOptions) {
+  url(
```

Comment:
Happy to revert.  I added this for myself while looking at `configuration.url()` and thought it might be useful to leave.